.github/workflows/docker-build-push.yml

```
@@ -26,7 +26,7 @@ jobs:
           images: bids/validator
 
       - name: Build and push Docker image
-        uses: docker/build-push-action@v4.0.0
+        uses: docker/build-push-action@v4.1.1
```

README just shows v4

```suggestion
        uses: docker/build-push-action@v4
```